Casks/l/lets-desktop-app.rb

```diff
@@ -0,0 +1,37 @@
+cask "lets-desktop-app" do
```

Comment:
```suggestion
  cask "lets"
```

Please also update the filename to `lets.rb`